src/dotnet-grpc/Commands/CommandBase.cs

```diff
@@ -315,7 +315,7 @@ internal string[] GlobReferences(string[] references)
                 var directoryToSearch = Path.GetPathRoot(reference)!;
                 var searchPattern = reference.Substring(directoryToSearch.Length);
 
-                var resolvedFiles = Directory.GetFiles(directoryToSearch, searchPattern);
+                var resolvedFiles = Directory.GetFiles(directoryToSearch, searchPattern, searchOption);
```

Comment:
It's not a good idea to import all proto files, it's better to go through the proto imports recursively and import the specified files if they exist. It will look like this: we add only the `securities.proto` file, the definition of the service, the rest used will be pulled up recursively.
example:
```
syntax = "proto3";
import "proto/fftapi/v1/security.proto";
package grpc.fftapi.v1;

message GetSecuritiesRequest {
}

message GetSecuritiesResult {
  repeated proto.fftapi.v1.Security securities = 1;
}

service Securities {
   rpc GetSecurities(grpc.fftapi.v1.GetSecuritiesRequest) returns (grpc.fftapi.v1.GetSecuritiesResult);
}
```
This example is from a real service. The approach that implements recursive directory search not only fails to complete successfully but also generates unnecessary services and types.